modules/default/calendar/calendarutils.js

```diff
@@ -296,38 +307,42 @@ const CalendarUtils = {
 					// Loop through the set of date entries to see which recurrences should be added to our event list.
 					for (let d in dates) {
 						let date = dates[d];
+						// FIXME: We now use node-ical instead of ical.js, but method stays the same.
```

Comment:
The comment says that we use(d) ical.js to parse the ical-file, which was changed in v 2.15. if I remember correctly.
Currently we are using `node-ical`, such that the comment is out of date but the method is still valid.
Maybe there is more elegant way using a built-in method, since the date variable represents a date object - but I have to double check this.
In both cases the comment has to be changed or removed but since this does not affect the issue and just served as a reminder.

Currently I'm reviewing the whole calendar module, making simplifications and refactoring such that I will have a second look on it :)

Reply:
The comment says that we use(d) ical.js to parse the ical-file, which was changed in v 2.15. if I remember correctly.
Currently we are using node-ical, such that the comment is out of date but the method is still valid.
Maybe there is more elegant way using a built-in method, since the date variable represents a date object - but I have to double check this.
In both cases the comment has to be changed or removed but since this does not affect the issue and just served as a reminder.

Currently I'm reviewing the whole calendar module, making simplifications and refactoring such that I will have a second look on it :)

Not quite sure where to reply, so just copy & paste the comment.
Sorry for any inconvenience :(

Reply:
So basically the comment should just be changed to say "node-ical" instead of "ical" , right?

Reply:
Ah, perfect. Thank you very much!
I'm going to update the PR in a few seconds :)